### PR TITLE
[FIX] mrp.workcenter xml view wrong field name

### PR DESCRIPTION
**Pull request by duybq86**
Description of the issue/feature this PR addresses: mrp.workcenter xml view wrong field name

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/mrp/views/mrp_workcenter_views.xml [hunk 1]
```diff
@@ -499,7 +499,7 @@
         <field name="model">mrp.workcenter.productivity</field>
         <field name="arch" type="xml">
             <search string="Operations">
-                <field name="workcenter_id"/>
+                <field name="production_id"/>
```
Comment (duong77476-viindoo) [inline addons/mrp/views/mrp_workcenter_views.xml:502]: Not wrong at all, if you want to add this field, create a custom module instead :v
And v14 is no longer support by odoo

Comment (duybq86) [reply addons/mrp/views/mrp_workcenter_views.xml:502]: What do u thing with the picture?
image

Comment (duong77476-viindoo) [reply addons/mrp/views/mrp_workcenter_views.xml:502]: @duybq86 Seem like they remove it in https://github.com/odoo/odoo/pull/63500 >=15.0
but still v14 is no longer support so i don't think they will merge this

Comment (duybq86) [reply addons/mrp/views/mrp_workcenter_views.xml:502]: odoo is community, and i think it good for some people, they can't upgrade to new version.

Comment (duong77476-viindoo) [reply addons/mrp/views/mrp_workcenter_views.xml:502]: I know, good luck bro

**Comment by Sehnde** [comment]
Hello,
Thank you for this PR but as duong77476-viindoo explained, support for 14.0 stopped almost a year ago and we are already refusing PRs for 15.0 due to the imminent release of 18.0. The only commits we will merge are for critical security flaws, so we will not merge this PR.
Thank you for your understanding and best regards,

**Comment by pfertyk** [comment]
Hi @duybq86 and thank you for your contribution! However, I'm closing this PR, since v14 is no longer supported (as explained above).

If the issue still happens in newer versions, please feel free to create another PR targeting the relevant branch :+1:

**Pull request closed without merging**